examples/simple-non-wasm/programs/counter/Forc.lock

```diff
@@ -0,0 +1,12 @@
+[[package]]
+name = 'core'
+dependencies = []
+
+[[package]]
+name = 'counter'
+dependencies = ['std git+https://github.com/fuellabs/sway?tag=v0.14.5#60c626cf486d5c53c44d84db1ec81cb90174cad3']
+
+[[package]]
+name = 'std'
+source = 'git+https://github.com/fuellabs/sway?tag=v0.14.5#60c626cf486d5c53c44d84db1ec81cb90174cad3'
+dependencies = ['core']
```

Comment:
It looks like this file was generated by forc `v0.14.5` - just a friendly reminder to update your installed `forc` version :)